src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/programsBuilder.js

```diff
@@ -172,7 +166,7 @@ async function getBuiltPrograms(
     locale: ?string,
 ) {
     if (cachedPrograms.length <= 0) {
-        return [];
+        return null;
```

Comment:
Changed that because flow was assuming that an empty array is the same as `Program`. 
We had this discussion the other day about `null` and `[]`. Seems that we could return `null` here and nothing much will change. 

What you think?

Reply:
I would prefer keeping this one as `[]`. I tried changing it back and I didn't see any flow errors? You could also remove the entire check I think (probably the best option)

Reply:
Will try it tomorrow thanks for looking into it

Reply:
Right no no errors. I am removing the statement as you suggested.